ocaml/idl/datamodel_types.ml

```diff
@@ -107,6 +107,16 @@ type api_release = {
   ; release_date: string option
 }
 
+let pp_api_release ppf
+    {code_name; version_major; version_minor; branding; release_date} =
+  Format.fprintf ppf
+    "{code_name=%a; version_major=%d; version_minor=%d; branding=%s; \
```

Comment:
I lack the context but this does not look like JSON. If it was meant to be, it would be better to create an association and let a JSON library create the string representation. Fine otherwise.

Reply:
Yes, please remove if it is not used in the code.